common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -470,6 +470,8 @@ enum class eChannelScanOperationCode : uint8_t {
     INVALID_PARAMS_CHANNELPOOL
 };
 
+enum class ePersistentParamBool : int8_t { DISABLE = 0, ENABLE = 1, NOT_CONFIGURED = -1 };
```

Comment:
Please add a doxygen comment explaining this type.
I don't understand its' purpose form the name. Bool suggests a TRUE/FALSE or ENABLED/DISABLED values, but you actually have 3 values.

```suggestion
enum class ePersistentParamBool : int8_t { NOT_CONFIGURED = -1, DISABLED = 0, ENABLED = 1 };
```

Reply:
It is not a bool, it is tri-state configuration.
So I would call it eTriStateConf or eConfEnabled.

Reply:
added Doxygen
reordered the values as suggested